ooniapi/services/ooniprobe/src/ooniprobe/routers/v2.py

```diff
@@ -65,6 +71,7 @@ def update_vpn_provider(db: Session, provider_name: str) -> models.OONIProbeVPNP
             openvpn_key=vpn_cert["key"],
         )
         db.add(provider)
+        vpn_endpoints = fetch_openvpn_endpoints()
```

Comment:
need try/except here too

Reply:
It's actually correct here to not wrap it try, except, since this is the initial case when we don't have anything in the database, so failing to fetch the endpoints is a hard failure.

Reply:
Maybe we should add a try, except block to the `        return update_vpn_provider(db, provider_name)
` call inside of `get_or_update_riseupvpn` where we could potentially return stale data in the event of an update failure.